packages/aws-lambda/src/triggers.js

```diff
@@ -69,9 +74,22 @@ function isApiGatewayProxyTrigger(event) {
   // NOTE: Gateway Rest API with lambda proxy -> always format 1.0
   // NOTE: Gateway HTTP API with lambda proxy -> default is 2.0, but can be configured on creation
   // NOTE: Gateway REST/HTTP API without lambda proxy -> format cannot be interpreted
+  // NOTE: Function URL also support version 2 and contains rawPath, Gateway HTTP API does not follow the format
```

Comment:
> Function URL also support version 2

Question: Is there a way to configure Function URL with format version 1.0? Or does Function Url only support 2.0?